### PR TITLE
Fix end date for Membership in free period offer 

### PR DESCRIPTION
**Pull request by jennysivapalan**
endDate on `user/me/details` is currently used in Guardian frontend to specify the Next Payment date on Membership details tab and End Date on the cancelled and tier change pages. For those who get a free period offer in Membership the endDate is displaying at 1 January 1970. 

I've updated the endDate on /user/me/details, if a payment has been made it is the charged through date. However if a user is in a free period then we can use the date of when the first payment will be which is the contract acceptance date.

I thought about creating a new field e.g. nextPaymentDate on user/me/details but the logic would be the same as endDate and then I'd have to update frontend too.

Before:
screen shot 2015-04-22 at 14 21 39
screen shot 2015-04-22 at 14 18 05

After:
screen shot 2015-04-22 at 14 24 09
screen shot 2015-04-22 at 14 16 50

:dancer:

### CODE REVIEW

File: frontend/app/controllers/User.scala [hunk 1]
```diff
@@ -22,6 +23,7 @@ trait User extends Controller {
   val casService = CASService
 
   def me = AjaxMemberAction { implicit request =>
+    println("hello")
```
Comment (mattandrews) [inline frontend/app/controllers/User.scala]: :wave:

Comment (jennysivapalan) [inline frontend/app/controllers/User.scala]: :wave: :earth_asia:

**Comment by davidrapson** [comment]
:no_entry_sign: :man: :date: :+1:

**Comment by jennysivapalan** [comment]
@jamesoram is okay to merge?

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**